install.sh

```
@@ -51,6 +51,73 @@ su -c "psql -c \"CREATE DATABASE mediacms\"" postgres
+    if bash "$NODEJS_SCRIPT"; then
+        export NVM_DIR="/root/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+        hash -r
+    else
+        echo "Error: Node.js installation script failed"; exit 1
+    fi
+else
+    echo "Warning: Could not install Node.js - install script not found"; exit 1
+fi
+    echo "Warning: Could not install Node.js - install script not found"; exit 1
+fi
```

Shellcheck (0.10.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)